src/fu-util.c

```diff
@@ -1456,6 +1456,7 @@ fu_util_device_emulate(FuUtilPrivate *priv, gchar **values, GError **error)
 {
 	g_autoptr(FuUtilDeviceTestHelper) helper = fu_util_device_test_helper_new();
 	helper->use_emulation = TRUE;
+	priv->flags |= FWUPD_INSTALL_FLAG_ONLY_EMULATED;
```

Comment:
should this behavior be a flag to the client?  Perhaps it makes sense for other commands too?

Reply:
Good idea; added.